vaadin-app-layout-flow-parent/vaadin-app-layout-flow/src/test/java/com/vaadin/flow/component/applayout/AppLayoutTest.java

```diff
@@ -179,6 +182,12 @@ public void setI18n() {
         assertEquals(i18n, systemUnderTest.getI18n());
     }
 
+    @Test
+    public void hasStyle() {
+        AppLayout appLayout = new AppLayout();
+        Assert.assertTrue(appLayout instanceof HasStyle);
```

Comment:
Let's reuse an already existing instance, no need to create a new one.

```suggestion
        Assert.assertTrue(systemUnderTest instanceof HasStyle);
```